apps/store/src/pages/api/session/create.ts

```diff
@@ -66,7 +66,7 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
   }
 
   const nextURL = new URL(ORIGIN_URL)
-  nextURL.pathname = PageLink.cart({ locale: DEFAULT_LOCALE })
+  nextURL.pathname = PageLink.cart({ locale: DEFAULT_LOCALE }).pathname
   const destination = nextURL.toString()
```

Comment:
this is awkward --- seems like we can just do:

```suggestion
  const destination = PageLink.cart({ locale: DEFAULT_LOCALE }).href
```

Reply:
True. Changing it.